fastlane/lanes/release.rb

```diff
@@ -278,6 +278,17 @@
     # Parse the provided version into an AppVersion object
     parsed_version = VERSION_FORMATTER.parse(new_version)
     previous_version = VERSION_FORMATTER.release_version(VERSION_CALCULATOR.previous_patch_version(version: parsed_version))
+    previous_release_branch = compute_release_branch_name(options: options, version: previous_version)
+
+    # Determine the base for the hotfix branch: either a tag or a release branch
+    base_ref_for_hotfix = if git_tag_exists(tag: previous_version)
```

Comment:
We might want to [also use `remote: true`](https://github.com/fastlane/fastlane/blob/master/fastlane/lib/fastlane/actions/git_tag_exists.rb#L32-L33) to check the tags on remote instead of just the ones fetched in the local working copy.

Especially thinking of the potential scenario where CI might do a lightweight checkout that hasn't fetched all the latest tags recently (`git fetch --tags`) and might only have older tags locally, not being aware that a tag that exists on the remote might potentially be absent locally if it hasn't been fetched since created.


```suggestion
    base_ref_for_hotfix = if git_tag_exists(tag: previous_version, remote: true)
```

Reply:
Good idea 👍